docs/vultr.md

```diff
@@ -0,0 +1,31 @@
+# Variomedia
+
+## Configuration
+
+### Example
+
+```json
+{
+  "settings": [
+    {
+      "provider": "vultr",
+      "domain": "example.com",
+      "owner": "potato",
+      "token": "AAAAAAAAAAAAAAA",
+      "ttl": 300,
+      "ip_version": "ipv4"
+    }
+  ]
+}
+```
+
+### Compulsory parameters
+
+- `"domain"` is the domain to update. It can be `example.com` (root domain) or `potato.example.com` (subdomain of `example.com`).
```

Comment:
Should the wildcard be documented as well, indicating the limitation on record type?

Reply:
Good shout, let me do that